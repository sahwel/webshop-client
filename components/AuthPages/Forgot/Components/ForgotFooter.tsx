import Button from '@/components/common/Form/Button/Button';
import Link from 'next/link';

const ForgotFooter = () => {
  return (
    <div>
      <Button children="Send email" type="submit" />
      <Link href="/login" passHref>
        <a className="text-center block text-sm mt-2 cursor-pointer">Back</a>
      </Link>
    </div>
  );
};

export default ForgotFooter;
