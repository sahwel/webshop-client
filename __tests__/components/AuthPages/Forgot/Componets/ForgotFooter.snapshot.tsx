import { render, renderHook, RenderHookResult } from '@testing-library/react';
import ForgotFooter from '@/components/AuthPages/Forgot/Components/ForgotFooter';

describe('ForgotFooter Snapshot', () => {
  it('renders ForgotFooter unchanged', () => {
    const { container } = render(<ForgotFooter />);
    expect(container).toMatchSnapshot();
  });
});
