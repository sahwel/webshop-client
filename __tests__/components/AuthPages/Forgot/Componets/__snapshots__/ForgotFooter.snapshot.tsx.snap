// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotFooter Snapshot renders ForgotFooter unchanged 1`] = `
<div>
  <div>
    <button
      class="button"
      type="submit"
    >
      Send email
    </button>
    <a
      class="text-center block text-sm mt-2 cursor-pointer"
      href="/login"
    >
      Back
    </a>
  </div>
</div>
`;
